quisp/modules/QRSA/HardwareMonitor/HardwareMonitor.cc

```diff
@@ -1144,6 +1348,43 @@ std::unique_ptr<NeighborInfo> HardwareMonitor::getNeighbor(cModule *qnic_module)
   return neighbor_info;
 }
 
+cModule *HardwareMonitor::getQNode() {
```

Comment:
remove this method and use `provider.getQNode`

Reply:
We may need mockQNode for HardwareMonitor_test?

Reply:
Added